Clients/GUI/ViewModels/SignUpViewModel.cs

```diff
@@ -1,5 +1,7 @@
 ﻿using System;
 using System.ComponentModel.DataAnnotations;
+using GUI.CustomValidationAtributes;
+using GUI.CustomValidationAttributes;
```

Comment:
Because of error in one of attributes files, we also have extra using here

Reply:
fixed in future commit